package.json

```diff
@@ -6,7 +6,6 @@
   "types": "index.d.ts",
   "scripts": {
     "test": "mocha --reporter spec --exit",
-    "pretest": "npm run lint",
```

Comment:
problem of that is then users running `npm test` won't get the lint errors
could you instead keep that and have a "mocha_test" script that would get call from the ci ?